auth/auth0_client.py

```diff
@@ -0,0 +1,147 @@
+import os
+
+import jwt
+import requests
+from auth.schemas import TokenData
+from dotenv import load_dotenv
+from fastapi import APIRouter, Request, Depends, HTTPException
+from fastapi.responses import RedirectResponse
+from jose import jwt, JWTError
+from main import oauth
+from settings import Settings
+from starlette.exceptions import HTTPException
+
+load_dotenv()
+
+AUTH0_DOMAIN = os.getenv("AUTH0_DOMAIN")
```

Comment:
move it to external file / settings.py / external class

Reply:
moved to external file settings.py and class Settings where crucial constants are taken from .env